plugins/autocomplete/plugin.js

```diff
@@ -207,6 +209,24 @@
 		 */
 		this._listeners = [];
 
+		/**
+		 * Template used to render selected dropdown match into an editor.
+		 *
+		 * You can use {@link CKEDITOR.plugins.autocomplete.model#data data item} properties to customize the template.
+		 *
+		 * ```javascript
+		 * config.outputTemplate = `<a href="/tracker/{ticket}">#{ticket} ({name})</a>`;
```

Comment:
I'd change `config.outputTemplate` to something else. Reason is that we use `config.xyz` for sampling global CKEditor configuration (`CKEDITOR.config`).

It might result with people trying passing `{ outputTemplate: 'foo' }` to the editor config and being surprised it doesn't work.

Unfortunately this case is tricky, as at this moment the template gets only passed though the constructor (cnosidering `autocomplete` plugin alone).